docs/_src/tutorials/tutorials/1.md

```diff
@@ -142,7 +142,7 @@ print(dicts[:3])
 document_store.write_documents(dicts)
 ```
 
-## Initalize Retriever, Reader,  & Pipeline
+## Initalize Retriever, Reader,  & Finder
```

Comment:
I am not sure about this. I would expect here Pipeline instead of Finder.

Reply:
Thanks for pointing that out, not sure how that snuck in. It's been reverted now